### PR TITLE
Add files_external:export command

### PR DESCRIPTION
**Pull request by icewind1991**
As alias of `occ files_external:list --output=json_pretty --show-password --full`

cc @PVince81

### CODE REVIEW

**Comment by mmattel** [comment]
Great, we can now export and import (Add import command for files external #21392)
Please add a documentation for both commands

**Comment by PVince81** [comment]
@icewind1991 did you test ?

```
PHP Fatal error:  Class 'OCA\Files_External\Command\Export' not found in /srv/www/htdocs/owncloud/apps/files_external/appinfo/register_command.php on line 44
PHP Stack trace:
PHP   1. {main}() /srv/www/htdocs/owncloud/occ:0
PHP   2. require_once() /srv/www/htdocs/owncloud/occ:11
PHP   3. OC\Console\Application->loadCommands() /srv/www/htdocs/owncloud/console.php:80
PHP   4. require() /srv/www/htdocs/owncloud/lib/private/console/application.php:73
```

**Comment by PVince81** [comment]
Had the old version. Re-fetched now.

**Comment by PVince81** [comment]
Tested, works :+1: 

Found another bug however: authentication_type is always "password" regardless of my selection.
Will raise separately.

**Comment by PVince81** [comment]
Raised here https://github.com/owncloud/core/issues/21771

File: apps/files_external/command/export.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+use OC\Core\Command\Base;
+use OC\User\NoUserException;
+use OCA\Files_external\Lib\StorageConfig;
+use OCA\Files_External\Service\BackendService;
+use OCA\Files_external\Service\GlobalStoragesService;
+use OCA\Files_external\Service\ImportLegacyStoragesService;
+use OCA\Files_external\Service\UserStoragesService;
+use OCP\IUserManager;
+use OCP\IUserSession;
```
Comment (MorrisJobke) [inline apps/files_external/command/export.php]: All of the above is not needed

Comment (MorrisJobke) [inline apps/files_external/command/export.php]: Only following is needed:

```

use Symfony\Component\Console\Input\ArrayInput;
use Symfony\Component\Console\Input\InputArgument;
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Output\OutputInterface;
```

Comment (icewind1991) [inline apps/files_external/command/export.php]: To much copy-paste, fixed

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Comment by MorrisJobke** [comment]
@carlaschroder This is a new command (it's an alias) to easily export mount point setups. This only works for the admin mount points. @icewind1991 Correct me if I'm wrong.

**Comment by PVince81** [comment]
I haven't pinged @carlaschroder yet because I wanted to write a summary of all new commands that will be available through "occ files_external". A few more will come.
See https://github.com/owncloud/core/issues/20368#issuecomment-157761007

**Comment by carlaschroder** [comment]
Thanks @PVince81 and @MorrisJobke, much appreciated. I'll wait for @PVince81.

**Pull request merged**